### DISCUSSION TITLE
Diploma thesis

### DISCUSSION THREAD

**Discussion by zdendai** [bcakmakoglu/vue-flow#91] [General]
Hi, I have a thesis assignment in which I wanted to use react-flow, but the company for which the work is to be done has a requirement to write the project in Vue so I was looking if there are alternatives for react-flow and I came across vue-flow. I would like to ask if the code is stable and ready enough to be used as well as react-flow and if the documentation and code won't change every week.

**Comment by bcakmakoglu** [comment]
Hi!

Thanks for your interest in vue flow 🚀 

The API isn't going to change in any major breaking way, i.e. I try to keep it backwards compatible with any 0.4.x changes already implemented.
The docs will be updated though, but mostly by adding new segments and info on how to accomplish certain things like dynamically adding more handles to a node or similar common issues people might face.
The examples section of the docs will be updated accordingly as well.

You will have to consider for yourself if that satisfies your constraints but I will gladly assist if you need any help 😄

**Reply by zdendai** [reply]
Thank you very much for your prompt reply. I'm trying out the library and it looks like it's good and easy to use, even seems a bit easier than the original react-flow. Vue3 + Typescript is new to me, so I'm happy to reach out to you and ask for help if needed. 

Thank you for your work and I wish you the best!
